src/flow/src/adapter/flownode_impl.rs

```diff
@@ -92,6 +95,35 @@ impl Flownode for FlowWorkerManager {
                     .map_err(to_meta_err)?;
                 Ok(Default::default())
             }
+            Some(flow_request::Body::Flush(FlushFlow {
+                flow_id: Some(flow_id),
+            })) => {
+                // TODO(discord9): impl individual flush
+                debug!("Starting to flush flow_id={:?}", flow_id);
+                // wait 100ms so that the source data is sended to our source
+                // so case like we run too quickly before input data have chance to be sended
+                // TODO(discord9): found better way to impl this flush
```

Comment:
If consistency is not guaranteed, this sleep is of little significance.

Reply:
> If consistency is not guaranteed, this sleep is of little significance.

useful in sqlness test, changed to using a rwlock anyway